bokeh/charts/models.py

```diff
@@ -143,7 +143,8 @@ def add_chart_index(self, data):
             n_rows = len(list(data.values())[0])
 
             # add composite chart index as column
-            data['chart_index'] = [self.label] * n_rows
+            # TODO: data['chart_index'] = [self.label] * n_rows
+            data['chart_index'] = [str(self.label)] * n_rows
```

Comment:
@fpliger, following up on the E-mail, there are some changes I made, but I doubt they make sense.